tests/utils/exec.js

```diff
@@ -23,7 +23,7 @@ export const createExec = (props: Props) => {
   TestRenderer.create(
     <Value initial={''}>
       {input => {
-        stateValue_ = input.value;
+        stateValue_ = props.format(input.value);
```

Comment:
why format here?

Reply:
Otherwise empty string is compared to `0.00` and fails